dags/utils/helpers.py

```diff
@@ -19,22 +19,26 @@ def get_postgres_uri(name):
     )
 
 
-def create_collector_task(name, dag, environment=None):
+def create_collector_task(name, dag, command=None, environment=None):
+    default_command = 'make start {}'.format(name)
+
     return _create_task(
         task_id='collector_{}'.format(name),
         dag=dag,
         image='okibot/collectors:latest',
-        command='make start {}'.format(name),
+        command=command or default_command,
```

Comment:
So the idea here was to call the helper with `command = 'euctr 2001-01-01'`. But this might be the more transparent way to do it.

Another thought would be to use Airflow variables, if they are present.

Say we do `args = args or variable_args` or something similar, where `variable_args` would be a `Variable.get('EUCTR_CMD_ARGS'). 

```python
variable_args = Variable.get('EUCTR_CMD_ARGS')
# [...]
args = args or variable_args
# and command would get the args appended
```